controllers/activemqartemis_reconciler.go

```diff
@@ -2495,13 +2501,40 @@ func (r *ActiveMQArtemisReconcilerImpl) createExtraConfigmapsAndSecretsVolumeMou
 					}
 				}
 			}
+
+			if strings.HasSuffix(secret, brokerPropsSuffix) {
+				bpSecret := &corev1.Secret{}
+				bpSecretKey := types.NamespacedName{
+					Name:      secret,
+					Namespace: r.customResource.Namespace,
+				}
+				if err := resources.Retrieve(bpSecretKey, client, bpSecret); err != nil {
+					return nil, nil, err
+				}
+
+				if len(bpSecret.Data) > 0 {
+					bpKeysMap := make(map[string]string)
+					for bpk := range bpSecret.Data {
+						bpKeysMap[bpk] = ""
+					}
+					for _, key := range sortedKeys(bpKeysMap) {
```

Comment:
bpKeysMap could be avoided by using `sortedKeysStringKeyByteValue`, i.e.
```suggestion
					for _, key := range sortedKeysStringKeyByteValue(bpSecret.Data) {
```

Reply:
+1